Update for Fall 2025 cohort (#23)
nmfs-openscapes/nmfs-openscapes.github.io: data-academy.qmd

You will receive a certificate for each of the Skills Paths that you complete, as well as for individual courses.

## Fall 2025 Cohort

The Fall Cohort is designed for those with some R experience, and is intented to be more self-directed. Participants can choose their own learning paths based on their interests and needs, and/or choose to follow the same [curriculum] as those in the Spring cohort. You can see the suggested curriculum schedule for the Fall cohort in the [curriculum spreadsheet](https://docs.google.com/spreadsheets/d/1YdE_khWiEr46fgdUyAG7GUg6fGBEAgQ9RRl9ac__0hQ).

**When:** Aug 18 - Dec 19, 2025. 

- Complete lessons on your own schedule.
- Wednesdays: Help desk with the Openscapes team alongside other participants (Optional, via Zoom).

> Particpants for the Fall cohort will be selected from the waitlist of the Spring cohort.

Reviewer: ```suggestion
Particpants for the Fall cohort were selected from the waitlist of the Spring cohort.
```
